docs/developer/extending/apps/local-apps-development.mdx

```diff
@@ -0,0 +1,132 @@
+
+We will alter behavior of the app, to be available on different hosts, depending "who is asking".
+
+- For browser, app will be available on http://localhost:3000
+- For Saleor in the container, app will be available on http://host.docker.internal:3000.
+
+:::note
+Read more about [host.docker.internal](https://docs.docker.com/desktop/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host) in Docker documentation.
+:::
+
+Saleor App Template ships with documented [`.env.example` file](https://github.com/saleor/saleor-app-template/blob/main/.env.example) you can rename to `.env`. It will be automatically loaded by the app.
+
+To override app's base URLs, you need to set following envs:
+
+```dotenv
+APP_IFRAME_BASE_URL=http://localhost:3000
+APP_API_BASE_URL=http://host.docker.internal:3000
+```
+
+Remember that `:3000` port if a default port Next.js uses for development. If you change it, you need to update the envs accordingly.
+
+You can see [the Manifest endpoint](https://github.com/saleor/saleor-app-template/blob/main/src/pages/api/manifest.ts) to see how the envs are used. In the nutshell:
+
+1. `appUrl` which Dashboard uses to open the app in the iframe is using "localhost".
+2. Other API endpoints are using `host.docker.internal` to ensure container can reach them. Saleor will call few of them (registration handshake, webhooks, fetching app logo...)
+
+:::caution
+You need to restart your app after changing the envs.
+:::
+
+At this point, your app will be still available on [http://localhost:3000](http://localhost:3000). You can open it in the browser and see if it works.
+You will see that manifest under [http://localhost:3000/api/manifest](http://localhost:3000/api/manifest) contains `host.docker.internal` URLs.
+
+### Installing App in Saleor
+
+The final step is install the app in Saleor. You can do that via API or the Dashboard. We will use the Dashboard:
+
+1. Open Dashboard ([http://localhost:9000](http://localhost:9000))
+2. Navigate to Apps section ([http://localhost:9000/apps](http://localhost:9000/apps))
+3. Click "Install external app" button and paste your `http://host.docker.internal:3000/api/manifest` URL.
+4. Continue with installation process.
+
+Your app should be soon available on the Apps' list. You can open it and see if it works.
+
+:::caution
+Be careful not to override envs in the deployment environment. By default, App will use the same endpoint it is reach from.
+Thanks to that, you can install App on a cloud like Vercel and use it's unique domains for each deployment.
+:::
+
+# Working on Linux
+
+This article uses workaround for Docker limitations on non-Linux systems. If you are working on Linux, you can leverage Docker's HOST network to achieve the same result.
+
+To do that, visit your Saleor Platform's `docker-compose.yml` file and change the network driver to `host`.
```

Comment:
@przlada can you confirm it?

Reply:
Yes, I confirm that

Reply:
Do we recommend that people fork the repository and modify the code?

Reply:
@patrys indirectly. Do you see other solutions? Of course devs can still use the non-Linux solution, but IMO its easier to just change the network

Reply:
@mmiszy have you actually used it? I tried and it wasn't working for me. But I haven't spent time on it really, just changed network to "host" If we add this to docs, I want to make sure it will work